src/Visualization/StackOffset.elm

```diff
@@ -0,0 +1,170 @@
+module Visualization.StackOffset
+    exposing
+        ( stackOffsetNone
+        , stackOffsetDiverging
+        , stackOffsetExpand
+        , stackOffsetSilhouette
+        , stackOffsetWiggle
```

Comment:
I think these could drop the `stackOffset` prefix, since that's already provided by the module name (at least for the internal names).